equinix/resource_fabric_service_profile.go

```diff
@@ -1133,40 +1128,40 @@ func marketingInfoToFabric(schemaMarketingInfo []interface{}) v4.MarketingInfo {
 	return marketingInfos
 }
 
-func processStepToFabric(processStepl []interface{}) []v4.ProcessStep {
-	if processStepl == nil {
+func processStepToFabric(processSteps []interface{}) []v4.ProcessStep {
+	if processSteps == nil {
 		return []v4.ProcessStep{}
 	}
-	var processSteps []v4.ProcessStep
-	for _, processStep := range processStepl {
-		psTitle := processStep.(map[string]interface{})["title"].(interface{}).(string)
-		psSubTitle := processStep.(map[string]interface{})["sub_title"].(interface{}).(string)
-		psDescription := processStep.(map[string]interface{})["description"].(interface{}).(string)
-		processSteps = append(processSteps, v4.ProcessStep{
+	var processStepRes []v4.ProcessStep
+	for _, pcs := range processSteps {
```

Comment:
We should use `index, processStep` for the range variables. Then inside the loop we should assign to `processStepsRes` by index. This matches the return type of `[]v4.ProcessStep`